### DISCUSSION TITLE
Возможность открывать двери крузера

### DISCUSSION THREAD

**Discussion by Darling-in-Balbes** [octothorp-team/public-dobrograd#641] [Предложения]
Предложение: предлагаю добавить возможность открывать двери полицейского крузера и использовать их в качестве укрытия. Открыть двери можно будет только передние при помощи команд: /openleftdoor и /openrightdoor, а также через С меню. Для открытия двери нужно быть около передней части авто или внутри самого авто. Двери будут работать как объект, т.е через них не будут проходить пули. 
Плюсы: поднятие атмосферы, ведь отныне не придется отыгрывать /me и /it. Ну и ,конечно же, это плюс копам, ведь за ними можно будет прятаться (логично).
Минусы: может немного подняться нагрузка на сервера.

**Comment by Leygun** [comment]
@Darling-in-Balbes 
Привет! 
В реалях сурса такое невозможно организовать. Для того, чтобы пули не проходили сквозь двери, необходимо будет менять коллизию автомобиля, что невозможно сделать.
